skglm/datafits/single_task.py

```diff
@@ -264,8 +289,16 @@ class Huber(BaseDatafit):
 
     Attributes
     ----------
+    delta : float
+        Shape hyperparameter.
```

Comment:
Can we be more detailed mathematically here ? I don't know what shape means in this context. Is is a scale?

Reply:
I've changed it to `Threshold` as it separates the domain where the function is quadratic and where it becomes linear (https://cvxr.rbind.io/cvxr_examples/cvxr_huber-regression/). Open to any suggestions